pkgs/development/python-modules/openusd/default.nix

```diff
@@ -153,7 +151,6 @@ buildPythonPackage rec {
 
   propagatedBuildInputs =
     [
-      boost
```

Comment:
this should stay in `propagatedBuildinputs`, otherwise downstream projects like blender will fail

Reply:
huh, colour me surprised! can confirm that it builds for me locally after adding that back